### ISSUE TITLE
[Feature Request] Zoom in and out buttons

### ISSUE THREAD

**Issue by zero77** [mozilla/lightbeam-we#241]
Can you please add two buttons to zoom in and out when in the graph view.
Thanks

**Comment by princiya** [comment]
Hi @zero77, thank you for requesting a feature.

We do support drag, zoom and pan in the existing version. [Reference](https://github.com/mozilla/lightbeam-we/pull/116)

**Comment by zero77** [comment]
Hi @princiya 
I was meaning more in and out, for if you have a lot in the graph and want a closer look or a look from further back.

**Comment by zero77** [comment]
@princiya 
Yes, I noticed that but, not all mouse pads support pinch and drag, that’s why I was asking for a butten.
